src/main/java/com/bart/scorebetlive442/controller/MatchController.java

```diff
@@ -0,0 +1,65 @@
+package com.bart.scorebetlive442.controller;
+
+import com.bart.scorebetlive442.mapper.MatchMapper;
+import com.bart.scorebetlive442.model.Match;
+import com.bart.scorebetlive442.model.json.MatchCreateJson;
+import com.bart.scorebetlive442.service.MatchService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/matches")
+public class MatchController {
+
+    private final MatchService matchService;
+    private final MatchMapper matchMapper;
+
+    @Autowired
+    public MatchController(MatchService matchService, MatchMapper matchMapper) {
+        this.matchService = matchService;
+        this.matchMapper = matchMapper;
+    }
+
+    @PostMapping(value = "/create")
+    public ResponseEntity<Match> createTeam(@RequestBody MatchCreateJson matchCreateJson) {
```

Comment:
we wszystkich kontrolerach brakuje przemapowania typu zwracanego (na tym przykładzie, jak do aplikacji wpada:
MatchCreateJson -mapujemy-> Match (strzelamy do serwisu i serwis coś zwraca/Match) to odpowiedź z Controllera też powinna być przemapowana (np. MatchResponseJson). Jeżeli na razie nie widzisz potrzeby na rozbijanie osobno MatchCreateJson, MatchResponseJson, MatchUpdateJson, etc. to użyjmy jednej klasy po prostu MatchJson (ale nie używajmy "modelowego" Match).

jako temat zaawansowany - zamiast tworzyć osobno warianty MatchCreateJson, MatchUpdateJson, MatchResponseJson można użyć widoków w Jacksonie (https://www.baeldung.com/jackson-json-view-annotation)